src/RemoteStorageService/Check.js

```diff
@@ -0,0 +1,24 @@
+import { useRemoteStorageMappings } from '@folio/stripes/smart-components';
+import { useHoldings } from '../providers';
+
+
+export const useByLocation = () => {
+  const remoteMap = useRemoteStorageMappings();
+
+  return ({ fromLocationId, toLocationId }) => (fromLocationId in remoteMap) && !(toLocationId in remoteMap);
```

Comment:
Could this be called with an `undefined` for either of the location ids? There're too many usages throughout the PR for me to track down completely, but it caught my eye. `undefined in remoteMap` will evaluate to `false`. The first thing that came to mind was situations where you could have some stale data or something.

It's legitimate to say that "no, that should never happen and it should spectacularly and loudly fail if it does." 😄

Reply:
The point here is to return `true` only if we move something from remote location to non-remote location.

- when `toLocationId` is `undefined` - it could be a black hole, but  it's not marked as remote, so
`(depends-on-from) && !(false)` === `depends-on-from`

- when `fromLocationId` is `undefined` - it certainly is not known remote location,
`(false) && !(whatever)` === `false`

- when both of them are `undefined` - the `fromLocationId` is undefined enough
`(false) && !(false)` === `false`